.github/workflows/test.yml

```
@@ -0,0 +1,209 @@
+name: Tests zds-site
```

On va croiser ce nom partout, et il prend de la place quand on a des tâches qu'ont de biens longs noms. De plus il n'est pas représentatif car en l'état ce _workflow_ ne fait pas que tester.

Ok, là c'est un détail.
```suggestion
name: CI
```